lib/udf_findhotel_utils.rb

```diff
@@ -10,22 +10,25 @@ class UdfFindhotelUtils
             if not room:
               return None
             else:
-              adults = 0
-              children = 0
-              rooms = room.split('|')
-              for r in rooms:
-                ch = r.split(':')
-                adults += int(ch[0])
-                if len(ch) > 1:
-                  children += len(ch[-1].split(','))
+              try:
```

Comment:
@CanCeylan There is no best case, it is rather about the behaviour you want to implement in your code.
About having several `try/except` block, it depends of the scope of code where you want to catch the exception. 
- If you want to cover a whole block, use one `try`. 
- If you want to cover some small pieces of your code located on different places put several `try`.

About using different `except` clauses it depends what you want to do with the exceptions caught :
- You need to apply the same code to several exceptions -> gather them in one `except` clause
- You need to process different exceptions in a different way -> use several `except` clauses`